test/unit/liquid/forms_test.rb

```diff
@@ -27,7 +27,9 @@ class Liquid::FormsTest < ActiveSupport::TestCase
   end
 
   test 'password_reset form' do
-    form = get('password_reset', "  ")
+    form = get('password_reset', 'site_account')
+    account = FactoryBot.create(:account)
+    form.context['site_account'] = Liquid::Drops::Message.new(account)
```

Comment:
Why is this a `Liquid::Drops::Message` instead of `Account` or maybe `Liquid::Drops::Account`

Reply:
After reading this https://github.com/3scale/porta/pull/1466#discussion_r373538298
It should be a `Liquid::Drops::Account`